tests/replication2_ssl.erl

```diff
@@ -188,6 +189,25 @@ confirm() ->
             ]}
     ],
 
+    SSLConfig8 = [
+        {riak_repl,
+            [
+                {fullsync_on_connect, false},
+                {fullsync_interval, disabled}
+            ]},
+        {riak_core,
+            [
+                {ssl_enabled, true},
+                {peer_common_name_acl, ["*.basho.com"]},
```

Comment:
Not sure why GitHub left this as a "valid" comment - SSLConfig8 now has no ACL, and I added 9 to make sure we tested wildcard certs with ACLs